### PR TITLE
Added Google home alarm sensor

### PR DESCRIPTION
**Pull request by eliseomartelli**
**Description:**
Added documentation for the googlehome alarm sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20709
## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/googlehome.markdown [hunk 1]
```diff
@@ -51,6 +55,10 @@ devices:
       description: Device types that will be tracked [see device types](#device_types), by default all types are tracked.
       required: false
       type: list
+    track_alarms:
+      description: Setting to tell the component to track the alarms of the device.
+      required: false
+      type: boolean
```
Comment (rohankapoorcom) [inline source/_components/googlehome.markdown:60]: Please add the default here

File: source/_components/googlehome.markdown [hunk 2]
```diff
@@ -12,14 +12,18 @@ ha_release: 0.88
 ha_category:
   - Hub
   - Presence Detection
+  - Sensor
 ha_iot_class: "Local Polling"
 redirect_from:
   - /components/device_tracker.googlehome/
+  - /components/sensor.googlehome/
```
Comment (rohankapoorcom) [inline source/_components/googlehome.markdown]: Did this page ever exist? If not, I don't think we need this.

File: source/_components/googlehome.markdown (hunk 1 above)
Comment (eliseomartelli) [reply source/_components/googlehome.markdown:60]: Took care of that now 👍

File: source/_components/googlehome.markdown (hunk 2 above)
Comment (eliseomartelli) [reply source/_components/googlehome.markdown]: ohh, right. Removed 😄

**Review by rohankapoorcom — APPROVED** [review]
Looks good to me! :)

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Think it is time to merge this PR, good work! 👍

**Pull request merged**